app/views/documents/edit.html.erb

```diff
@@ -48,12 +45,10 @@
           Upload documents
         </button>
 
-        <%= link_to next_path, class: "button" do %>
-          I'll do this later
-        <% end %>
+        <button class="button button--next button--cta button--full-width" type="submit">
+          Done uploading documents
```

Comment:
ah ok. I guess they could click the "back" button in the nav, but I see your point